main/commands/combineVideos.ts

```diff
@@ -10,22 +10,19 @@ export async function combineVideos(
   outputFilePath: string,
   notifyEvent?: EventEmitter
 ): Promise<void> {
-  winston.info('Generating videoList file');
-  notifyEvent && notifyEvent.emit('Generating videoList file');
-  tmp.setGracefulCleanup();
-  // Create a temporary list file to concatenate videos (see https://trac.ffmpeg.org/wiki/Concatenate)
-  const { path: file, cleanup } = await tmp.file({ postfix: '.txt' });
-  const contents = videoPaths.map((videoPath) => `file '${videoPath}'`).join('\n');
-  writeFileSync(file, contents);
   winston.info('Combining Videos');
   notifyEvent && notifyEvent.emit('Combining Videos');
-  const combineProcess = spawnSync(
-    paths.ffmpeg,
-    ['-f', 'concat', '-loglevel', 'error', '-safe', '0', '-i', file, '-c', 'copy', outputFilePath],
-    {
-      stdio: 'pipe',
-    }
-  );
+  const args: string[] = ['-v', 'error'];
```

Comment:
In the context of this PR, I'd like you to explain the how/why you've made these changes.  For example, you deleted the `-loglevel error -safe 0` and replaced it with `-v error` but without an explanation and going back to the docs, I'm not confident that they are equivalent (they _seem_ like they could be equivalent...)

Reply:
`-v` is just shorthand for `-loglevel` (https://www.ffmpeg.org/ffmpeg.html#Generic-options) and `-safe 0` is an option specific to the old concat demuxer command (https://ffmpeg.org/ffmpeg-formats.html#Options).

Reply:
perfect, this is the explanation I was looking for.